watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1690/amo_1690_case_study.yaml

```diff
@@ -0,0 +1,52 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.065
+    units: USD_2020/kWh
+  biogas:
+    value: 0.4
+    units: USD_2020/m^3
+  fertilizer:
+    value: 0.5
+    units: USD_2020/kg
+global_parameters:
+  plant_lifetime:
+    value: 20
+    units: year
+  utilization_factor:
+    value: 0.9
+    units: dimensionless
+  land_cost_percent_FCI:
+    value: 0
+    units: dimensionless
+  working_capital_percent_FCI:
+    value: 0
+    units: dimensionless
+  salaries_percent_FCI:
+    value: 0
+    units: 1/year
+  benefit_percent_of_salary:
+    value: 0
+    units: dimensionless
+  maintenance_costs_percent_FCI:
+    value: 0.02
+    units: 1/year
+  laboratory_fees_percent_FCI:
+    value: 0
+    units: 1/year
+  insurance_and_taxes_percent_FCI:
+    value: 0
+    units: 1/year
+  wacc:
+    # Weighted Average Cost of Capital (WACC)
+    value: 0.08
```

Comment:
```suggestion
    value: 0.05
```

Reply:
This is just the discount rate, right? 0.05 is usually our default, but this team gave us some TEA/LCA data that lists 0.08. So I used that instead.